package.json

```
@@ -35,5 +35,8 @@
     "@vercel/ncc": "^0.38.1",
     "eslint": "^8.52.0",
     "jest": "^29.7.0"
+  },
+  "resolutions": {
+    "cross-spawn": "^7.0.6"
```

should we narrow this down to only 7.0 versions?
```suggestion
    "cross-spawn@7.0.*": "^7.0.6"
```

---

we only source 7.0 currently so i think its fine

---

the issue is that this isn't something we directly install, its a dependency for other packages we consume so upgrading those may cause this resolution to kick in inadvertently in the future and cause unexpected behaviour.. but its fine for now

---

Fair

---

I don't think resolutions like this work for yarn `1.22.10`, at least it didn't when I tried on this repo.

---

well.. that's probably something we should address in a separate PR 😅